### DISCUSSION TITLE
아이템 먹으면 효과음 나오도록 할 수 있나요?

### DISCUSSION THREAD

**Discussion by PH62mem** [naverz/zepeto-studio-kor#919] [5. Q&A]
아이템 먹으면 효과음 나오도록 할 수 있나요?

bgm말고 먹었을때만 2초정도 나오고 다시 다른 아이템 먹으면 또 나오고

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 효과음도 유니티와 동일한 방법으로 구현이 가능합니다.
아이템과 충돌했을때 AudioSource.PlayOneShot()를 하는 스크립트를 추가해서 구현할 수 있습니다. 

아래는 시작 시에 효과음을 재생 시키는 예제 코드입니다.

public audioSource: AudioSource;
public testSfx: AudioClip;
Start() {    
        this.audioSource.PlayOneShot(this.testSfx);
 }
